backend/src/main/java/com/woowacourse/friendogly/member/dto/request/SaveMemberRequest.java

```diff
@@ -1,7 +1,12 @@
 package com.woowacourse.friendogly.member.dto.request;
 
+import jakarta.validation.constraints.Email;
 import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Size;
 
-public record SaveMemberRequest(@NotBlank String name) {
+public record SaveMemberRequest(
+        @NotBlank @Size(max = 15) String name,
+        @NotBlank @Email String email
+) {
```

Comment:
검증에 실패했을 때를 대비해 message를 추가해도 되겠네요.
이 경우 어노테이션이 길어질 것 같으니 개행을 하는 건 어떤가요?

```suggestion
public record SaveMemberRequest(
        @NotBlank(message = "빈 문자열이나 null을 입력할 수 없습니다.")
        @Size(max = 15, message = "닉네임은 1글자 이상 15글자 이하여야 합니다.")
        String name,
        
        @NotBlank(message = "빈 문자열이나 null을 입력할 수 없습니다.")
        @Email(message = "올바른 email 형식이 아닙니다.")
        String email
) {
```

Reply:
인정합니당~ 
적절한 메시지를 함께 응답해주는게 재시도를 유도하는데 좋겠네요 😊